src/sdk/pynni/nni/nas/pytorch/fixed.py

```diff
@@ -33,6 +34,32 @@ def __init__(self, model, fixed_arc, strict=True):
             raise RuntimeError("Unexpected keys found in fixed architecture: {}.".format(fixed_arc_keys - mutable_keys))
         if mutable_keys - fixed_arc_keys:
             raise RuntimeError("Missing keys in fixed architecture: {}.".format(mutable_keys - fixed_arc_keys))
+        self._fixed_arc = self._convert_human_readable_architecture(self._fixed_arc)
+
+    def _convert_human_readable_architecture(self, human_arc):
+        result_arc = {k: to_list(v) for k, v in human_arc.items()}  # there could be tensors, numpy arrays, etc.
+        # First, convert non-list to list.
+        result_arc = {k: v if isinstance(v, list) else [v] for k, v in result_arc.items()}
```

Comment:
why not combine line40 and line42 by creating a list for non list in `to_list` function?

Reply:
I don't think so. `to_list` is to be convert non-python lists (e.g., numpy array or tensors) to python lists without affecting the semantics. In line 42, it changes single value to a list of one single value. If I merge it into `to_list`, `to_list` might be confusing to whoever tries to use it in their own code.

Reply:
btw, I've made it more clear in comments.